src/figuresClasses.ts

```diff
@@ -1,11 +1,101 @@
-export interface Figure {}
+export interface Figure {
+  shape: `triangle` | `circle` | `rectangle`;
+  color: `red` | `green` | `blue`;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle';
+
+  color: 'red' | 'green' | 'blue';
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
+    a: number,
+    b: number,
+    c: number,
+  ) {
+    if (a === 0 || b === 0 || c === 0) {
+      throw new Error('Sides can not be equal to zero');
```

Comment:
[CODE STYLE] - The error message 'Sides can not be equal to zero' could be more descriptive. It's better to provide additional context to the error message to make it clear why the sides cannot be zero. For example: 'Triangle sides must be greater than zero.'

Reply:
a